### PR TITLE
Issue #5423: Add web server to Telemetry.

### LINKED ISSUE #5423: [DX] Telemetry: Add Web Server (Apache/nginx/etc)
## Description of the need

In addition to PHP version, MySQL version, and Server OS, we should collect web server: Apache or nginx (supported platforms) as well as other options (either consolidated into "Other" or separated).

## Proposed solution

Add Web Server into `system_telemetry_data()`.

Part of meta issue https://github.com/backdrop/backdrop-issues/issues/5231.

### PR DESCRIPTION
**Pull request by quicksketch**
Fixes https://github.com/backdrop/backdrop-issues/issues/5423

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

File: core/modules/telemetry/telemetry.telemetry.inc [hunk 1]
```diff
@@ -23,6 +23,11 @@ function telemetry_telemetry_info() {
     'description' => t('The operating system of your server, such as RedHat Linux, Debian, Windows, etc.'),
     'project' => 'backdrop',
   );
+  $info['web_server'] = array(
+    'label' => t('Web Server'),
```
Comment (klonos) [inline core/modules/telemetry/telemetry.telemetry.inc]: Does this need to be Title Case?

```suggestion
    'label' => t('Web server'),
```

Comment (quicksketch) [reply core/modules/telemetry/telemetry.telemetry.inc]: Officially yes I think it's supposed to be title case.

**Comment by laryn** [comment]
Thanks @quicksketch for filing and fixing, and @klonos for the review! Merged into `1.x` and `1.20.x`.

**Pull request merged**